docs/UserGuide.md

```diff
@@ -222,9 +224,9 @@ Adds an elderly, a volunteer, or a pairing between one elderly and one volunteer
 
 Adds an elderly to FriendlyLink.
 
-Format: `add_elderly n/NAME eic/NRIC bd/BIRTH_DATE [p/PHONE] [e/EMAIL] [a/ADDRESS] [re/REGION] [r/RISK_LEVEL] [t/TAG]… [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…`
+Format: `add_elderly n/NAME ic/NRIC bd/BIRTH_DATE [re/REGION] [r/RISK_LEVEL] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]… [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…`
 
-* Every elderly must have a unique `NRIC`, which is 9-alphabets long.
+* Every elderly must have a unique `NRIC`, which is 9-alphabets long, and conforms to the format `1 letter - 7 numbers - 1 letter`.
```

Comment:
Since we already specified the requirements in the earlier part, we can instead create a hyperlink that goes to section where we specify the requirements instead. Same for `add_volunteer`